Simplified/NYPLMyBooksDownloadCenter.m

```diff
@@ -456,6 +457,15 @@ - (void)deleteLocalContentForBookIdentifier:(NSString *const)identifier account:
       [[AudiobookFactory audiobook:json] deleteLocalContent];
       break;
     }
+    case NYPLBookContentTypePDF: {
+      NSError *error = nil;
+      if (![[NSFileManager defaultManager]
+          removeItemAtURL:[self fileURLForBookIndentifier:identifier account:account]
```

Comment:
As long as the PDFs are indeed saved here, it seems good. Ideally (as we talked about), they would have a `.pdf` file extension instead, but that's a much bigger thing to change if we already have PDFs saved this way.